src/po/ja.po

```diff
@@ -17,10 +17,10 @@ msgstr ""
 "POT-Creation-Date: 2018-05-01 10:46+0900\n"
 "PO-Revision-Date: 2017-05-18 00:45+0900\n"
 "Last-Translator: MURAOKA Taro <koron.kaoriya@gmail.com>\n"
-"Language-Team: vim-jp (https://github.com/vim-jp/lang-ja)\n"
-"Language: Japanese\n"
+"Language-Team: Japanese <https://github.com/vim-jp/lang-ja>\n"
+"Language: ja\n"
```

Comment:
For Japanese translation, please send a PR to https://github.com/vim-jp/lang-ja .
Is there any reason you changed `(...)` to `<...>`?
Should the language be `ja` instead of `Japanese`?

Reply:
https://www.gnu.org/software/gettext/manual/gettext.html#Header-Entry

Yeah "Language:" is the short language code https://www.gnu.org/software/gettext/manual/gettext.html#Language-Codes like "ja" or "ja_XX" for both language code and country code.

Maybe (...) can be used to but it seems <...> are used for links like in "Last-Translator:".

Reply:
Does changing the one ja.po make both ja.euc-jp.po, ja.po and ja.sjis.po?
I also suggested (...) to <...> in the comment not that it matter. Just a suggest to use the same style.